languages/csharp/README.md

```diff
@@ -1,17 +1,65 @@
-# Requirements
+# Bitwarden Secrets Manager SDK
 
-- Dotnet
+.NET bindings for interacting with the [Bitwarden Secrets Manager]. This is a beta release and might be missing some functionality.
 
-# Installation
+## Create access token
 
-From the `languages/csharp/` directory,
+Review the help documentation on [Access Tokens]
 
-```bash
-dotnet restore
+## Usage code snippets
+
+### Create new Bitwarden client
+
+```csharp
+const string accessToken = "<access-token>";
+using var bitwardenClient = new BitwardenClient();
+bitwardenClient.AccessTokenLogin(accessToken);
+```
+
+### Create new project
+
+```csharp
+const string organizationIdStr = "<organization-id>";
+var organizationId = Guid.Parse(organizationIdStr);
+var projectResponse = bitwardenClient.Projects().Create(organizationId, "TestProject");
+```
+
+### List all projects
+
+```csharp
+var response = bitwardenClient.Projects.List(organizationId);
+```
+
+### Update project
+
+```csharp
+var projectId = projectResponse.Id;
+projectResponse = bitwardenClient.Projects.Get(projectId);
+projectResponse = bitwardenClient.Projects.Update(projectId, organizationId, "TestProjectUpdated");
+```
+
+### Add new secret
+
+```csharp
+var key = "key";
+var value = "value";
+var note = "note";
+var secretResponse = bitwardenClient.Secrets.Create(key, value, note, organizationId, new Guid[]{projectId});
+var secretId = secretResponse.Id;
 ```
 
-# Run
+### List secrets
 
-```bash
-dotnet run
+```csharp
+var secretIdentifiersResponse = bitwardenClient.Secrets.List(organizationId);
 ```
+
```

Comment:
We're missing an example on updating secret.

Reply:
added an example on updating secret.